curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/clean-up-your-project-with-modules.english.md

```diff
@@ -8,7 +8,7 @@ forumTopicId: 301549
 ## Description
 <section id='description'>
 
-Right now everything you have is in your server.js file. This can lead to hard to manage code that isn't very expandable.
+Right now, everything you have is in your server.js file. This can lead to hard to manage code that isn't very expandable.
```

Comment:
```suggestion
Right now, everything you have is in your <code>server.js</code> file. This can lead to hard to manage code that isn't very expandable.
```